test/NuGet.Core.Tests/NuGet.Commands.Test/ProjectResolutionTests.cs

```diff
@@ -165,13 +159,7 @@ public async Task ProjectResolution_ProjectReferenceWithInCompatibleTFM()
 
                 var logger = new TestLogger();
 
-                var restoreContext = new RestoreArgs()
-                {
-                    Sources = new List<string>() { packageSource.FullName },
-                    GlobalPackagesFolder = packagesDir.FullName,
-                    Log = logger,
-                    CacheContext = new SourceCacheContext()
-                };
+                var restoreContext = new RestoreArgs() {Sources = new List<string>() {packageSource.FullName}, GlobalPackagesFolder = packagesDir.FullName, Log = logger, CacheContext = new SourceCacheContext()};
```

Comment:
I don't like this change, I think it's much easier to read one property per line.  It's also unrelated to the title of this PR.